app/controllers/MovementsController.scala

```diff
@@ -116,4 +195,7 @@ class MovementsController @Inject()(
             InternalServerError(s"Failed with the following error: $e")
         }
   }
+
+  def get(arrivalId: ArrivalId): Action[AnyContent] = defaultActionBuilder(_ => NotImplemented)
```

Comment:
We've added this for the `GET /movements/arrival/{arrivalId}` so that it will resolve in the reverse router. This is used in the `Location` header on a successful `POST /movements/arrival`